packages/block-editor/src/components/block-settings-menu/block-settings-dropdown.js

```diff
@@ -33,7 +33,10 @@ const POPOVER_PROPS = {
 };
 
 function CopyMenuItem( { blocks, onCopy } ) {
-	const ref = useCopyToClipboard( () => serialize( blocks ), onCopy );
+	const ref = useCopyToClipboard(
+		() => serialize( blocks, { retainCopyAttributes: false } ),
```

Comment:
`retainCopyAttributes` isn't a spectacular name, but I had difficulty coming up with a better one. `excludeNonCopyableAttributes`? `retainOnlyCopyableAttributes`?

Reply:
Maybe `includeCopyableAttributes`? Also let's add `__experimental` for the moment 😀

Reply:
Agreed about `__experimental` :)

Updated to `__experimentalExcludeNonCopyableAttributes` (and inverted the logic). It's an... intense name 😅 but I'm having trouble being more concise. I'm worried that something like `retainCopyAttributes` or `includeCopyableAttributes` doesn't really get across that it means retain _only_ copyable attributes.